lk2nd/device/dts/msm8909/apq8009w-wtp.dts

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	model = "Qualcomm APQ8009W";
+	qcom,msm-id = <QCOM_ID_APQ8009 0>, <QCOM_ID_APQ8009W 0>;
+	qcom,board-id = <QCOM_BOARD_ID_MTP 0x105>;
+};
+
+&lk2nd {
+	mobvoi-catfish {
+		model = "Mobvoi TicWatch Pro (WF12096)";
+		compatible = "mobvoi,catfish";
+		lk2nd,dtb-files = "mobvoi,catfish";
```

Comment:
This looks very wrong and is not a usual format for the dtb filenames. If you don't know, or there is not yet a canonical dtb name for the device, this better be a "fixme" comment like on few other devices

Reply:
[catfish-stock-boot-dts.zip](https://github.com/user-attachments/files/16571596/catfish-stock-boot-dts.zip)
I'm not sure what you are trying to ask, but I uploaded the stock dts (decompiled already) if you like to take a look. There are no ref's no mobvoi, so it may need a FIXME.

Reply:
the lk2nd,dtb-files prop is for booting mainline linux with dtb selected by lk2nd. There is no "qcom/mobvoi,catfish.dts" in mainline and this shouldn't be a correct name. If you already have/plan to make a mainline dts, use that name; otherwise put a comment like this:

https://github.com/msm8916-mainline/lk2nd/blob/fa2a7615bfaa458caf528b5489eb6f032485bd7e/lk2nd/device/dts/msm8916/msm8916-asus-z00e.dts#L15

Reply:
Ok, I did a FIXME, but is the skeleton64.dtsi alright? Or should it be skeleton32.dtsi?

Reply:
doesn't currently matter but prefer correct one, so 32 for 32bit soc